frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/pushdialog/GitPushDialog.java

```diff
@@ -61,42 +64,39 @@ public GitPushDialog(
         Registry.is("ide.perProjectModality") ? IdeModalityType.PROJECT : IdeModalityType.IDE);
     this.isForcePushRequired = isForcePushRequired;
     this.project = project;
-    this.pushAction = new PushSwingAction();
-    this.pushAction.putValue(DEFAULT_ACTION, Boolean.TRUE);
-
-    var dialog = new MyVcsPushDialog(project, selectedRepositories.asJava(), pushSource) {
-      @Override
-      @UIEffect
-      public @Nullable VcsPushOptionValue getAdditionalOptionValue(PushSupport support) {
-        return GitPushDialog.this.getAdditionalOptionValue(support);
```

Comment:
@mkondratek note that this still wasn't correct and only passed thanks to another (or maybe the same?) bug in CF... `GitPushDialog.this` isn't `@Initialized` when `MyVcsPushDialog` is instantiated :/

Reply:
https://github.com/typetools/checker-framework/issues/3408